concepts/05 Widgets/MultiView/10 Switch Between Views.md

```diff
@@ -1,16 +1,23 @@
 By default, an end user can switch between views by swiping. Assign **false** to the [swipeEnabled](/api-reference/10%20UI%20Widgets/dxMultiView/1%20Configuration/swipeEnabled.md '/Documentation/ApiReference/UI_Widgets/dxMultiView/Configuration/#swipeEnabled') option to disable this feature.
 
 ---
+
 ##### jQuery
 
     <!--JavaScript-->
-    $(function() {
-        $("#multiViewContainer").dxMultiView({
-            dataSource: multiViewItems,
+    $(function () {
+        $('#multiViewContainer').dxMultiView({
+            dataSource: [
```

Comment:
There's no need in a data source if you are showing an option that doesn't depend on it.